doc/rtd/reference/cli.rst

```diff
@@ -3,6 +3,17 @@
 CLI commands
 ************
 
+Cloud-init ships multiple binaries that are intended for user interaction.
+
+These binaries are:
```

Comment:
These are all scripts, so it feels odd to call these binaries. Perhaps "executables" would make more sense here?